### PR TITLE
Report errors to Sentry

### PR DESCRIPTION
**Pull request by phuctm97**
This installs Sentry recommended configuration for a Vue web application. It enables Sentry `production`, `lrc`, and `testing`.

Changes:

- Install Sentry.
- Add a flag to enable report by default, saved to localStorage at `torus-enable-crash-reporter`.
- Add crash report section to WalletSettings.
- Mutate flag locally and over API to `torus-backend` when toggling then enabling/disabling Sentry accordingly.

### CODE REVIEW

**Comment by phuctm97** [comment]
The newly-added CI code is commented out to avoid unexpected run (not sure how it is exactly configured yet). If it looks good enough, I'll uncomment then.

File: src/components/WalletSettings/CrashReport/CrashReport.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+<script>
+export default {
+  name: 'CrashReportSettings',
+  computed: {
+    enabled: {
+      get() {
+        return this.$store.state.crashReport
+      },
+      set(value) {
+        this.$store.dispatch('setCrashReport', Boolean(value))
```
Comment (chaitanyapotti) [inline src/components/WalletSettings/CrashReport/CrashReport.vue:19]: we use `mapActions` and `mapState`  syntax in the project.

File: public/sw.js [hunk 2]
```diff
@@ -875,7 +875,7 @@ self.addEventListener('notificationclick', function (e) {
 
 self.__precacheManifest = [
   {
-    url: '/js/app.js',
+    url: '/v1.9.5/js/app.js',
```
Comment (chaitanyapotti) [inline public/sw.js:878]: Undo this pls

File: src/components/WalletSettings/CrashReport/CrashReport.vue (hunk 1 above)
Comment (phuctm97) [reply src/components/WalletSettings/CrashReport/CrashReport.vue]: Updated.

File: public/sw.js (hunk 2 above)
Comment (phuctm97) [reply public/sw.js]: Reverted.

**Review by chaitanyapotti — APPROVED** [review]
lgtm. Great work

**Pull request merged**